### ISSUE TITLE
notebooks don't give the same visibility to errors as data explorer does

### ISSUE THREAD

**Issue by barbaranelson** [influxdata/ui#3153]
Here was my flux script:

```
import "http"

http.post(url:  "https://us-central1-1.gcp.cloud2.influxdata.com/api/v2/scripts/086381b08d86c000/invoke",
  headers: {
    Accept : "application/csv", 
    "Content-Type": "application/json",
    Authorization : "Token MYTOKENGOESHERE"
    }
)
```
The problem with the script is that http.post only returns the post response code (200, 404 etc.) and not the response body, so there was nothing to visualize in the table.The error I got in notebooks (in the validation panel) was:
`error @4:5-4:18: expected [A] but found int (argument tables)`

The error I got in data explorer was: 
`error in query specification while starting program: this Flux script returns no streaming data. Consider adding a "yield" or invoking streaming functions directly, without performing an assignment`

The data explorer error was a lot more useful to try to figure out what was going on.

**Comment by jzahnd** [comment]
Will discuss at grooming to follow up.

**Comment by jzahnd** [comment]
@jzahnd will reach out to the Flux team in reference to this issue

**Comment by jzahnd** [comment]
@rockstar will hopefully be taking a look at this today

**Comment by rockstar** [comment]
> these errors come from the api, so it's not an issue with the parser. we should get the flux teams input on what to do here

Which api are we talking about? I'm unfamiliar with the source here, but we're obviously following two different code paths. Could you help me understand what those two code paths are?

**Comment by jzahnd** [comment]
Jessica will follow up with Flux team about queryd issue.

**Comment by nathanielc** [comment]
What I think is happening is that notebooks is just pasting in the user content with the limit code which creates some odd looking Flux code which in turn creates an odd error.

For example the above script gets turned into this:

```
import "http"

http.post(url:  "https://us-central1-1.gcp.cloud2.influxdata.com/api/v2/scripts/086381b08d86c000/invoke",
  headers: {
    Accept : "application/csv", 
    "Content-Type": "application/json",
    Authorization : "Token MYTOKENGOESHERE"
    }
)
  |> limit(n:100)

```


But `http.post` doesn't return a table but rather an integer. So its like calling running this Flux code `limit(tables: 1, n: 100)`. Flux complains that the argument to tables isn't a table but rather an integer.

```
error @4:5-4:18: expected [A] but found int (argument tables)
```

The `[A]` is how we represent tables in the type system (i.e. a list of records). We can likely improve that bit.

So TL;DR adding `|> limit(n:100)` without understanding the user's code causes a new error to be reported before the error about not returning streaming data can be hit.

**Comment by barbaranelson** [comment]
@drdelambre can we drop the |> limit(n:100) in flux panels for now, and let the user deal with it if they would get too large a response?  They can add a limit themselves if they need to.  Then we can address this more comprehensively in the query work that you and Nathanial will be doing.

**Comment by jzahnd** [comment]
We may want to table this while we have our Query Experience discussion?
